topic-controller/src/main/java/io/strimzi/controller/topic/Controller.java

```diff
@@ -707,7 +710,7 @@ void onTopicCreated(TopicName topicName, Handler<AsyncResult<Void>> resultHandle
             @Override
             public void handle(Future<Void> fut) {
 
-                TopicMetadataHandler handler = new TopicMetadataHandler(vertx, kafka, topicName) {
+                TopicMetadataHandler handler = new TopicMetadataHandler(vertx, kafka, topicName, new BackOff(config.get(Config.TOPIC_METADATA_MAX_ATTEMPTS))) {
```

Comment:
We should factor out a method to encapsulate the creating of the `BackOff`, rather than repeating this code in two places.

Reply:
Done.